packages/blade/src/components/NavigationBar/_decisions/decisions.md

```diff
@@ -0,0 +1,261 @@
+# NavigationBar - One Top Navigation Bar
+
+The navigation bar is positioned at the top of the screen that provides quick access to different products, search & user profile. 
+
+<img src="./top-nav-bar.png" width="100%" />
```

Comment:
What is that heavy vertical border in this image next to Razorpay logo?

Reply:
Bug while exporting the figma image.